bin/Create_RoutingTable.py

```diff
@@ -0,0 +1,903 @@
+#!/bin/python
```

Comment:
this looks fine. to improve the code quality, please add tests for your new features (any topology and routing tables) in the tests folder!